cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/update_munge_key.sh.erb

```diff
@@ -15,12 +15,31 @@ MUNGE_USER="<%= @munge_user %>"
 MUNGE_GROUP="<%= @munge_group %>"
 SHARED_DIRECTORY_COMPUTE="<%= @shared_directory_compute %>"
 SHARED_DIRECTORY_LOGIN="<%= @shared_directory_login %>"
+CONFIG_PROCESS="False"
```

Comment:
I would propose a variable called `DISABLE_RESOURCE_CHECK`, which is false by default, and in case the boolean flag is used, it is set to true.

If this variable is true, then the part of the check on the compute (and login) fleet status is skipped.

If we go for `DISABLE_RESOURCE_CHECK`, then I would use the option `d` in the `getopts` below.

Reply:
Done!